components/builder-jobsrv/src/data_store.rs

```diff
@@ -165,7 +165,7 @@ impl DataStore {
                                         FOR UPDATE SKIP LOCKED
                                         LIMIT $1
                                     LOOP
-                                        UPDATE jobs SET job_state='Dispatched', updated_at=now() WHERE id=r.id RETURNING * INTO r;
+                                        UPDATE jobs SET job_state='Dispatched', scheduler_sync=false, updated_at=now() WHERE id=r.id RETURNING * INTO r;
```

Comment:
Now that we've deployed to acceptance, we can't edit functions anymore, now we need to create a new version and have the server call that. We could destroy the database and recreate it just before we go live so we clear all the old funcs